src/server/handlers/launchpad.js

```diff
@@ -301,3 +301,57 @@ export const completeSnapAuthorization = async (req, res) => {
     });
   });
 };
+
+export const getSnapBuilds = (req, res) => {
+  const snap_link = req.query.snap_link;
+
+  if (snap_link) {
+    return getLaunchpad().get(snap_link).then(snap => {
+      // TODO: bartaz: currently just limiting to 10 recent builds
+      // we will need to process it properly later for paging/loading more
+      return getLaunchpad().get(snap.builds_collection_link, { start: 0, size: 10 })
+        .then(builds => {
+          return res.status(200).send({
+            status: 'success',
+            payload: {
+              code: 'snap-builds-found',
+              message: builds.entries
```

Comment:
My intention was for "code" to be a unique identifier for an outcome that could be matched to an error message to be shown to the user. The "message" was to be a human-readable alternative to describe what happened to a developer reading the network logs in devtools. These two fields (with the status field) are fine for describing errors. "code" might even be sufficiently readable on its own.

I think we should pass builds as its own property in this case.